src/components/Modals/ContactFormModal.jsx

```diff
@@ -43,9 +45,25 @@ const ContactFormModal = ({
             Alerter.error('error.save')
           }
         }}
-        onCancel={onClose}
       />
     </ModalDescription>
+    <ModalFooter className="u-ta-right">
+      <Button
+        type="button"
+        theme="secondary"
+        label="cancel"
+        onClick={onClose}
+      />
+      <Button
+        type="submit"
+        label="save"
+        onClick={() =>
+          document
```

Comment:
maybe I should add a comment here to explain why we need this?

Reply:
Yep, I want an explanation about this direct dom manipulation. Because there is no way that we merge that ^^

Reply:
Ok so I'll explain it here first :-) 
We need to be able to move the form buttons outside of the react-final-form because we need them in the modal footer so that they are sticky.

There are two ways to do this according to RFF documentation / examples : https://codesandbox.io/s/1y7noyrlmq Either we use a closure (I wanted to do this at first but we need a unique component with the modal and the form so... no) either we get the form id and submit it like I did here. Another solution would be that the form contains the modal components but I don't think it is a good idea. So it's not ideal but like this the form itself is still isolated and we will be able to use it for the myself page.

Reply:
now we use `_submitContactForm` variable for this